invokeai/app/services/shared/sqlite_migrator/migrations/migration_14.py

```diff
@@ -0,0 +1,61 @@
+import sqlite3
+
+from invokeai.app.services.shared.sqlite_migrator.sqlite_migrator_common import Migration
+
+
+class Migration14Callback:
+    def __call__(self, cursor: sqlite3.Cursor) -> None:
+        self._create_style_presets(cursor)
+
+    def _create_style_presets(self, cursor: sqlite3.Cursor) -> None:
+        """Create the table used to store model metadata downloaded from remote sources."""
+        tables = [
+            """--sql
+            CREATE TABLE IF NOT EXISTS style_presets (
+                id TEXT NOT NULL PRIMARY KEY,
+                name TEXT NOT NULL,
+                preset_data TEXT NOT NULL,
+                is_default BOOLEAN DEFAULT FALSE,
```

Comment:
Yeah good call, will update. I may call it `type` or something different because I think we will want a try concept of `category` here (and probably with workflows too, hmm)